app/templates/components/layer-palette.hbs

```diff
@@ -17,6 +17,7 @@
     <container.layer-group-toggle
       data-test-toggle-tax-lots
       @layerGroup={{this.layerGroups.tax-lots}}
+      @activeTooltip={{zoom-dependent-label this.layerGroups.tax-lots this.zoomWarningLabel this.mainMap.zoom}}
```

Comment:
This is great!

Do you think it would be possible to set this up in the contextual component that is yielded from the layer group container? See here: https://github.com/NYCPlanning/labs-zola/blob/develop/app/templates/components/labs-ui/layer-groups-container.hbs#L8-L15

That way, there's way less copy pasta... unless there's a _lot_ of variety across these components. I think it would save some overhead. You could definitely pass the other dependent arguments into the container.

Reply:
Ohhh good idea. Yeah I definitely thought about yielding it from `layer-group-container` since I thought it should be a sibling component/helper of `layer-group-toggle`, instead of wrapping it or being nested in it. I just didn't think of using one sibling component as a "helper" for another! I can give refactoring a go. 

I'm also starting to think maybe `layer-group-toggle` could have a parameter for a condition or function condition for showing the tooltip. I am cautious about expanding the `layer-group-toggle` api too much, though